### PR TITLE
DM-47038: Add pinhole, none, OG590_2_5mm filters for ComCam.

### PR DESCRIPTION
**Pull request by ktlim**

### CODE REVIEW

File: python/lsst/obs/lsst/filters.py [hunk 1]
```diff
@@ -443,14 +443,15 @@ def addFilter(filter_dict, band, physical_filter):
...
     physical_filter = f"{band}_{sn[3:]}"
-    lsstCamFilter = [f for f in LsstCamFiltersBaseline if f.band == band][0]
-
     addFilter(ComCamFilters_dict, band, physical_filter)
 
 
 ComCamFilters = [
     FilterDefinition(band="white", physical_filter="empty"),
     FilterDefinition(band="unknown", physical_filter="unknown"),
+    FilterDefinition(band="white", physical_filter="none"),
```
Comment (timj) [inline python/lsst/obs/lsst/filters.py:452]: Don't we use `NoFilterCollection` for this definition?

File: python/lsst/obs/lsst/filters.py [hunk 2]
```diff
@@ -443,14 +443,15 @@ def addFilter(filter_dict, band, physical_filter):
...
+    FilterDefinition(band="white", physical_filter="pinhole"),
+    FilterDefinition(physical_filter="OG590_2_5mm"),
```
Comment (timj) [inline python/lsst/obs/lsst/filters.py:454]: No band? What type of filter is this?

Comment (ktlim) [reply python/lsst/obs/lsst/filters.py:454]: It's some kind of narrowband filter.

Comment (timj) [reply python/lsst/obs/lsst/filters.py:452]: Looks like, from HSC, we would name the band OG590 then or something similar. (depending on what OG means)

File: python/lsst/obs/lsst/filters.py (hunk 1 above)
Comment (ktlim) [reply python/lsst/obs/lsst/filters.py]: Ah, yes, that looks better, changed to using it.

**Review by timj — APPROVED** [review]

**Pull request merged**